silkrpc/commands/eth_api.cpp

```diff
@@ -116,7 +116,7 @@ asio::awaitable<void> EthereumRpcApi::handle_eth_syncing(const nlohmann::json& r
 asio::awaitable<void> EthereumRpcApi::handle_eth_get_block_by_hash(const nlohmann::json& request, nlohmann::json& reply) {
     auto params = request["params"];
     if (params.size() != 2) {
-        auto error_msg = "invalid eth_getLogs params: " + params.dump();
+        auto error_msg = "invalid get_block_by_hash params: " + params.dump();
```

Comment:
Kudos for the correction! We should stick to API name in error messages, so `eth_getBlockByHash` -> `get_block_by_hash`

Reply:
see next two comments

Reply:
Sorry, I've switched the strings here. You should use `eth_getBlockByHash`

Reply:
ok